packages/eslint-plugin-jest/docs/rules/no-skipped-tests.md

```diff
@@ -0,0 +1,33 @@
+# Disallow Skipped Tests (no-skipped-tests)
+
+Jest has a feature that allows you to skip tests by appending `.skip` or prepending `x` to a test-suite or a test-case.
+Sometimes tests are skipped as part of a debugging process and aren't intended to be committed. This rule reminds you to remove .skip or the x prefix from your tests.
+
+## Rule Details
+
+This rule looks for every `describe.skip`, `it.skip`, `test.skip`, `xdescribe`, `xit` and `xtest` occurrences within the source code.
```

Comment:
According to the docs it is: http://facebook.github.io/jest/docs/api.html#the-jest-global-environment 
Is that out of date?